composer.json

```diff
@@ -16,7 +16,7 @@
 	},
 
 	"require": {
-		"php": "^5.3|^7.0"
+		"php": "^5.4|^7.0"
```

Comment:
This is a BC break, because you're increasing minimal supported PHP version for no reason.